### PR TITLE
Update tmux-cssh to 1.0.6-0

### PR DESCRIPTION
**Pull request by zchee**

### CODE REVIEW

File: Library/Formula/tmux-cssh.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 class TmuxCssh < Formula
   homepage 'https://github.com/dennishafemann/tmux-cssh'
-  url 'https://github.com/dennishafemann/tmux-cssh/archive/0.1.tar.gz'
-  sha1 '477a9079775ce48afe0c90699fa69f368e69cdfd'
+  url 'https://github.com/dennishafemann/tmux-cssh/archive/1.0.6-0.tar.gz'
+  sha1 '1c5dd181b9525766cc21b317c68c34cd69324b95'
```
Comment (mikemcquaid) [inline Library/Formula/tmux-cssh.rb]: Needs a `version "1.0.6-0"` line.

**Comment by zchee** [comment]
Fixed.

File: Library/Formula/tmux-cssh.rb [hunk 2]
```diff
@@ -1,9 +1,10 @@
 require 'formula'
 
 class TmuxCssh < Formula
+  version "1.0.6-0"
```
Comment (mikemcquaid) [inline Library/Formula/tmux-cssh.rb]: Please put this below `sha1`, thanks.

**Comment by asparagui** [comment]
Thanks!

**Pull request closed without merging**